src/main/groovy/com/blackducksoftware/integration/hub/detect/bomtool/npm/NpmConstants.groovy

```diff
@@ -0,0 +1,7 @@
+package com.blackducksoftware.integration.hub.detect.bomtool.npm
+
+class NpmConstants {
```

Comment:
These should be moved into NpmBomTool. We dont typically create separate classes for static strings